integrated_channels/api/v1/degreed/views.py

```diff
@@ -0,0 +1,41 @@
+"""
+Viewsets for Degreed
+"""
+
+from edx_rbac.mixins import PermissionRequiredMixin
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from rest_framework import permissions, viewsets
+from rest_framework.authentication import SessionAuthentication
+
+from django.core.exceptions import PermissionDenied
+from django.http import Http404
+
+from enterprise.models import EnterpriseCustomerUser
+from integrated_channels.degreed.models import DegreedEnterpriseCustomerConfiguration
+
+from .serializers import DegreedConfigSerializer
+
+
+class DegreedConfigurationViewSet(PermissionRequiredMixin, viewsets.ModelViewSet):
+    authentication_classes = (JwtAuthentication, SessionAuthentication,)
+    serializer_class = DegreedConfigSerializer
+    permission_classes = (permissions.IsAuthenticated,)
+    permission_required = 'enterprise.can_access_admin_dashboard'
+
+    def get_queryset(self):
+        enterprise_customer_users = EnterpriseCustomerUser.objects.filter(user_id=self.request.user.id)
```

Comment:
I assume you dont have to check for user existence here before asking for user.id because this will never allow a null request.user because of the super classes?

Reply:
Since at the end of this whole thing you know you only want 1 or 0 enterprise customer UUID,  there's a util here you could use that only does 1 db hit, I think. See utils.get_enterprise_customer_user()

Reply:
yeah they'd get a 403 long before we get here if there was no user.